pkgs/development/python-modules/aiohttp/default.nix

```diff
@@ -67,7 +66,6 @@ buildPythonPackage rec {
     pytest-mock
     pytestCheckHook
     re-assert
```

Comment:
How about
```suggestion
    re-assert
  ] ++ lib.optional stdenv.isLinux [
    trustme
  ];
```
?

Reply:
Agreed, trustme looks very much like an optional dependency and this is the correct solution.

https://github.com/aio-libs/aiohttp/blob/cc6dc0c49f5d002485f9a3cdf9bc3127a3ac1388/tests/conftest.py#L14-L22

Reply:
Ah, interesting. I checked `setup.cfg` and found no sign of it, and saw that the tests still passed. But I didn't go into the tests themselves.

I've added the same test as in `pyopenssl`, with a comment so folks can re-add this one on darwin when it is fixed.